zipline/pipeline/factors/technical.py

```diff
@@ -144,6 +150,238 @@ def compute(self, today, assets, out, close, volume):
         out[:] = nanmean(close * volume, axis=0)
 
 
+class RollingPearsonOfReturns(SingleInputMixin, CustomFactor):
+    """
+    Calculates the Pearson product-moment correlation coefficient of the
+    returns of the given asset with the returns of all other assets.
+
+    **Default Inputs:** [Returns]
+
+    Parameters
+    ----------
+    baseline_asset : zipline.assets.Asset
+        The asset to correlate with all other assets.
+    returns_length : int > 1
```

Comment:
> or enforce that it's greater than 2.

The constraint on returns is greater than or equal to 2, so should I write this as `>=2`? I think "greater than 1" is correct.